Sprint-3/4-stretch-investigate/card-validator.test.js

```diff
@@ -0,0 +1,32 @@
+// Implement a function cardNumberValidator
```

Comment:
This is a good set of tests, but your function could currently just be:

```js
function cardNumberValidator(number) {
  return false;
}
```

And all the tests would pass. Can you work out why? How would you fix this?